index.js

```diff
@@ -59,14 +59,48 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
-    if (element.style.color === "green") {
-        element.style.color = "black"
+header.addEventListener('click', function(title){
+    // console.log(title)
+    if (title.target.style.color === "purple") {
+        title.target.style.color = "pink"
     } else {
-        element.style.color = "green"
+        title.target.style.color = "purple"
     }
-}
+})
+
 
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector('#profile button.like-button')
+// const likes = document.querySelector("#profile > p")
+
+likeButton.addEventListener('click', function(likes){
+    const likeCounter = document.querySelector("#profile > p")
+    console.log(likes)
+    traveler.likes+=1
```

Comment:
You were on the right track by defining our likes `<p>` tag on line 75! Because remember - we don't want this information to persist and update in our traveler object.

If i separated this out into its own function it might look something like this:

```javascript
const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling;
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```

Reply:
(ps here's more on [previousElementSibling](https://developer.mozilla.org/en-US/docs/Web/API/Element/previousElementSibling)